NMS-14621: Quick Start - Visualize Data (#5353)
OpenNMS/opennms: docs/modules/operation/pages/quick-start/visualize-data.adoc

= Step 9: Visualize Data

This section describes how to visualize your environment's data (using the dashboard, graphs, and so on) in {page-component-title}.

== {page-component-title} home page

The {page-component-title} home page presents an overview of important network status metrics, as soon as you log in.
By default, it displays pending situations and problems, ongoing outages, 24-hour availability statistics, and a map displaying real-time regional status data.

When you log in, the visualizations on the home page are not filtered.
You can apply filters to the data using the sections on the right side of the screen (Resource Graphs, KSC Reports, and Quick Search).

For more information, see xref:operation:deep-dive/admin/webui/grafana-dashboard-box.adoc[Grafana Dashboard Box] in the Deep Dive section.

[[qs-visualize-dashboard]]
== Dashboard

You can view the default dashboard by clicking menu:Dashboards[Dashboard] in the top menu bar.

> The dashboard compiles environment data to generate an overview of monitored components; this information can often be a useful starting point for more detailed analysis of issues.

Author: I used it here to mean "data collected from your monitored network." Do we have a better term for this?

> The dashboard compiles environment data to generate an overview of monitored components; this information can often be a useful starting point for more detailed analysis of issues.

Reviewer: Maybe say "network environment data" to be explicit.

= Step 9: Visualize Data

This section describes how to visualize your environment's data (using the dashboard, graphs, and so on) in {page-component-title}.

== {page-component-title} Home page

> The {page-component-title} Home page presents an overview of important network status metrics, right after you log in.

Reviewer: ```suggestion
The {page-component-title} home page presents an overview of important network status metrics, as soon as you log in.
```
I'd like to minimize the capitalization we use.

By default, it displays pending situations and problems, ongoing outages, 24-hour availability statistics, and a map displaying real-time regional status data.

When you log in, the visualizations on the Home page are not filtered.
You can apply filters to the data using the sections on the right side of the screen (Resource Graphs, KSC Reports, and Quick Search).

For more information, see xref:operation:deep-dive/admin/webui/grafana-dashboard-box.adoc[Grafana Dashboard Box] in the Deep Dive section.

[[qs-visualize-dashboard]]
== Dashboard

You can view the default dashboard by clicking menu:Dashboards[Dashboard] in the top menu bar.

> It compiles environment data to generate an overview of monitored components; this information can often be a useful starting point for more detailed analysis of issues.

Reviewer: ```suggestion
The dashboard compiles environment data to generate an overview of monitored components; this information can often be a useful starting point for more detailed analysis of issues.
```

> The dashboard is comprised of five components:

Reviewer: ```suggestion
The dashboard is comprised of the following components:
```
saves us updating numbers if the components ever change.

* *Surveillance view:* lets you model a monitoring context on the dashboard.
+
You can reduce visible information by selecting rows, columns, and cells to focus on, quickly limiting the data you have to sift through.
The rest of the dashboard updates dynamically.

* *Alarms:* shows unacknowledged alarms which should be escalated.
* *Notifications:* shows outstanding and unacknowledged notifications.
* *Outages:* shows all ongoing network outages.
* *Resource graphs:* shows time series reports for performance diagnosis.

For more information on each of the components, see xref:operation:deep-dive/admin/webui/dashboard.adoc[Dashboard] in the Deep Dive section.

[[qs-visualize-ops-board]]
== Operator board

> An operator board (ops board) helps you visualize monitoring information by configuring dashlet to display different kinds of data (alarms, maps, KSC reports, and so on).

Reviewer: ```suggestion
An operator board (ops board) helps you visualize monitoring information by configuring dashlets to display different kinds of data (alarms, maps, KSC reports, and so on).
```

Each dashlet visualizes specific information.
You can combine dashlets to build an overview of your environment that best suits your needs.

> Ops boards can be created and configured only by users with admin permissions.

Reviewer: ```suggestion
Only users with admin permissions can create and configure ops boards.
```

For more information, see xref:operation:deep-dive/admin/webui/surveillance-view.adoc[Surveillance View] in the Deep Dive section.

== Heatmap

You can use the heatmap to display either unacknowledged alarms or ongoing node outages.
Each visualization can be applied to categories, foreign sources, or node services.

You can view the heatmap by clicking menu:Status[Heatmap] in the top menu bar.

For more information, see xref:operation:deep-dive/admin/webui/heatmap.adoc[Heatmap] in the Deep Dive section.

== Trend

The Trend page displays small, inline charts of database-related statistics (nodes experiencing outages, unacknowledged alarms, and so on).
You can view it by clicking menu:Status[Trend] in the top menu bar.

For more information, see xref:operation:deep-dive/admin/webui/trends.adoc[Trend] in the Deep Dive section.

== Beyond Quick Start

Refer to xref:operation:deep-dive/admin/webui/introduction.adoc[Administrative Web Interface] in the Deep Dive section for more information on visualizations, including the following:

* xref:operation:deep-dive/admin/webui/opsboard/criteria-builder.adoc[Filtering dashlet information using the Criteria Builder].

> * xref:operation:deep-dive/admin/webui/ui-preview.adoc[Using the UI preview].

Reviewer: Let's keep this for now, but we may need to remove it as part of the H31 work to get rid of the UI preview and incorporate those screens into the product.
